src/main/java/org/reactome/release/qa/check/CoV2InfectionPathwayEventCheck.java

```diff
@@ -0,0 +1,188 @@
+            if (cov2Event.getAttributeValue(ReactomeJavaConstants.inferredFrom) == null && !hasRecentLiteratureReference(cov2Event)) {
+                issues.add("Does not contain a 2020 literature reference");
+            }
+            // Summation text was updated near end of release cycle where CoV-1-to-CoV-2 projections happened.
+            // If it didn't have the updated text, there are two reasons for it, outlined below.
+            if (!hasCorrectSummationFormat(cov2Event)) {
+                if (cov2Event.getAttributeValue(ReactomeJavaConstants.inferredFrom) == null) {
+                    issues.add("InferredFrom is empty but instance has COVID inference summation text");
+                } else {
+                    issues.add("InferredFrom is populated but instance does NOT have COVID inference summation text.");
+                }
+            }
+            if (issues.size() > 0) {
+                report.addLine(getReportLine(cov2Event, String.join("|",issues)));
+            }
+        }
+        report.setColumnHeaders(getColumnHeaders());
+        return report;
+    }
+
+    /**
+     * Checks if instance created dateTime is before most recent modified instance's dateTime. Also checks if
+     * any modified instance is present -- it should be given that there was at least 1 required modification from the
+     * CoV-1-to-CoV-2 merge.
+     * @param cov2Event - GKInstance, Event contained within 'SARS-CoV-2 Infection' Pathway.
+     * @return - boolean, true if modification property looks correct, false if not.
+     * @throws Exception, thrown by MySQLAdaptor.
+     */
+    private boolean hasRecentlyModifiedSummation(GKInstance cov2Event) throws Exception {
+       for (GKInstance summation : (Collection<GKInstance>) cov2Event.getAttributeValuesList(ReactomeJavaConstants.summation)) {
+            GKInstance createdInst = (GKInstance) summation.getAttributeValue(ReactomeJavaConstants.created);
+            List<GKInstance> modifieds = summation.getAttributeValuesList(ReactomeJavaConstants.modified);
+            if (modifieds.size() > 0) {
+                // At time of writing (August 2020), nothing is reported from this clause, but it is a good check.
+                GKInstance mostRecentModifiedInst = modifieds.get(modifieds.size() - 1);
+                // Only check Year/Month/Day since the changes took place more than a day before Curators got to look at them in gk_central.
+                LocalDate createdDateTime = getDateTimeFromInstance(createdInst);
+                LocalDate modifiedDateTime = getDateTimeFromInstance(mostRecentModifiedInst);
+                if (createdDateTime.isAfter(modifiedDateTime)) {
+                    return false;
+                }
+            // Should be at least 1 modified instance in CoV-1-to-CoV-2 instances.
+            } else {
+                return false;
+            }
+       }
+        return true;
```

Comment:
You could write this in a declarative rather than imperative way by doing something like:
```
return summations
.stream()
.allMatch(summation -> hasModifiedInstance(summation) && !createdDateAfterModifiedDate(summation))
```
Not a blocker - just a thought/suggestion.

Reply:
These tests are 4 months old at this point, and have pretty much been used as much as they will be. I think for posterity it would be better to keep it in the style when it was used

Reply:
Okay, no problem - I just like the stream style using `allMatch` and `anyMatch` in these situations where you're in a loop returning true or false.  It allows for concisely focusing on the match condition with one exit point.